### DISCUSSION TITLE
巴莱大厦的灵异故事

### DISCUSSION THREAD

**Discussion by kaijinti** [share121/inter-knot#936] [其他]
![20240810191202](https://github.com/user-attachments/assets/00312677-9dd1-4dda-a0b4-39c577c15e28)
之前就总是看见看见巴莱大厦的灵异传说，今天路过看见一个妹子不知道盯着什么在看，怪吓人的，她不会真能看见什么吧？

**Comment by fu-fu404** [comment]
这个委托我接了

**Comment by gaojiaobei** [comment]
哈人，听说巴莱大厦那里还有脏东西，那里的委托还是别接了

**Comment by wwwwsn147** [comment]
真的假的？！我就在巴莱大厦附近居住，那我晚上可不敢出门了：（

**Comment by Tsukinoamane** [comment]
不是芭莱大厦嘛（派蒙吃惊.jpg）

**Comment by SWYH34827** [comment]
有谁要一起去探险吗😉

**Comment by Mu-7593** [comment]
据说这里的【再好好想想】雕像附近，一到晚上就会出现诡异的黑影，不知道是不是真的
